books/free-programming-books-uk.md

```diff
@@ -1,5 +1,6 @@
 ### Index
 
+* [C and C++](#c-and-c++)
```

Comment:
can't use `+` in fragments. I think `c-and-c` works; look at other lists for examples

Reply:
Thank you, I found how anchors for c++ were built in other files, so I have specified it like "#c-and-cpp" in my case.